common/workunit/workunit.cpp

```diff
@@ -1830,6 +1847,7 @@ mapEnums workunitSortFields[] =
    { WUSFwuidhigh, "@" },
    { WUSFwildwuid, "@" },
    { WUSFappvalue, "Application" },
+   { WUSFfilewritten, "Files/File/@name" },
```

Comment:
Although these fields are 'workunitSortFields' fields,I think WUSFfilewritten is used as a filter (and probably others are too), but does it make sense to sort by WUSFfilewritten ?
What does it mean if there are > 1 Files/File/@name per workunit?